LabelGenerator.py

```diff
@@ -615,22 +633,13 @@ def main() -> None:
     # ############################################################################
     # Put your own resistor values in here!
     #
-    # This has to be either a 2D grid or a 1D array.
+    # This has to be either a 2D grid or a 1D array. The E12, E24, and E24_ALL
+    # constants can be passed to the generate_decades function to create typical
+    # resistor value sets.
     #
     # Add "None" if no label should get generated at a specific position.
     # ############################################################################
-    resistor_values: ResistorList = [
-        [0,            0.02,         .1],
-        [1,            12,           13],
-        [210,          220,          330],
-        [3100,         3200,         3300],
-        [41000,        42000,        43000],
-        [510000,       None,         530000],
-        [6100000,      6200000,      6300000],
-        [71000000,     72000000,     73000000],
-        [810000000,    820000000,    830000000],
-        [9100000000,   9200000000,   3300000000],
-    ]
+    resistor_values: ResistorList = [0] + generate_decades(E24, 0, 6) + [ 1000000, None, 2000000 ]
```

Comment:
I'd prefer to keep the original code here. Maybe one active and one commented out with an explanation that this is an alternative way to do it?

Remember, most people that ist this script have only a very small understanding of programming, so it's like to keep adjusting the values as easy as possible. Your change will stop some people from using this because they won't be able to figure out what the 0 and 6 mean.

So I think id like to keep my original version as the active one and add yours in a comment after or before.

Reply:
That is a good point.  I've updated the PR.